README.md

```diff
@@ -54,17 +56,17 @@ Some reviews will require 2 👍🏼, based on the characteristics of the review
 
 When a second reviewer is added, it's expected that this reviewer will be a member of a senior engineering group. The sr-eng review may add other reviewers as per their discretion.
 
-### Automated Checks 
+### Automated Checks
 
-PRs are expected to pass the following
+PRs are expected to pass the following across the organization
 
 - Linter
 - Docblock check
 - Unit test coverage
-- CI 
+- CI
+- PR Line size check
```

Comment:
what is this? It's about the indentation or it's about PRs smaller than X lines of code?

Reply:
Prs smaller or larger then x lines